tests/test_loader.py

```diff
@@ -42,7 +42,7 @@ def test_load_should_create_object_from_file_with_relative_segments():
     obj = m3u8.load(playlists.RELATIVE_PLAYLIST_FILENAME)
     expected_key_abspath = '%s/key.bin' % os.path.dirname(base_uri)
     expected_key_path = '../key.bin'
-    expected_ts1_abspath = '%s/entire1.ts' % base_uri
+    expected_ts1_abspath = '/entire1.ts'
```

Comment:
I think files with a leading `/` were being mis-handled as well.

On a file system, joining `/tmp/path/to/somewhere` with `/entire.ts` should produce `/entire.ts`.

```python
>>> from os.path import join as python_join
>>> python_join('/path/to/somewhere', '/entire1.ts')
'/entire1.ts'
```

Reply:
Agreed